1.setup.sh

```diff
@@ -48,7 +48,7 @@ sudo apt update
 function init_zsh() {
     sudo apt install zsh
     chsh -s $(which zsh)
-    cp ./default.zshrc ~/.zshrc
+    mv ./default.zshrc ~/.zshrc
```

Comment:
this is clean but I was thinking that this is brought local by the snippet in the README and should probably be cleaned up there too.